src/db/z_db.erl

```diff
@@ -233,7 +243,7 @@ assoc_props_row(Sql, Parameters, Context) ->
 
 
 %% @doc Return property lists of the results of a query on the database in the Context
```

Comment:
Line 245 is too long: %% @doc Return property lists of the results of a query on the database in the Context.